camayoc/tests/qcs/test_credentials.py

```diff
@@ -152,10 +232,17 @@ def test_negative_create_no_name(cleanup):
     :expectedresults: Error is thrown and no new host credential is created.
     :caseautomation: notautomated
     """
-    pass
+    client = api.Client(api.echo_handler)
+    cred = HostCredential(
+        client,
+        username='',
+        password=uuid4(),
+    )
+    response = cred.create()
+    assert response.status_code == 400
+    assert HostCredential().list().json() == []
```

Comment:
One thing I was thinking about was that if we ever want to run these in parallel we can't make assertions like this, because we do not know what other tests may be doing at that time.

Basically, I think it is fine if we want to just make the decision to run in serial, but if we want to leave the door open to running in parallel, I think it is better not to build up debt like this because it may be hard to root out later on

Reply:
This is a really good point, will improve it.